frontend/src/shared/modules/default-filters/components/default-filters.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <span class="text-gray-500 text-sm"> {{ copy }}</span>
+</template>
+
+<script setup lang="ts">
+import { IncludeEnum } from '@/modules/member/config/saved-views/settings/common/types/IncludeEnum';
+import { computed } from 'vue';
+import config from '../config/index';
+
+const props = defineProps<{
+    module: 'member' | 'organization',
```

Comment:
I wouldnt include anymore which module this component is present, i would just pass filter config there to have it more dynamic and not dependent on module